io_scene_niftools/modules/nif_export/armature/__init__.py

```diff
@@ -110,4 +113,26 @@ def export_bone_flags(self, b_bone, n_node):
                     # default for Div 2 final bones
                     n_node.flags = 0x0196
             else:
-                n_node.flags = 0x0002  # default for Morrowind bones
\ No newline at end of file
+                n_node.flags = 0x0002  # default for Morrowind bones
+
+    def export_lag_bone(self, b_bone, n_node):
+        """Exports a BSLagBoneController for the bone"""
+
+        #BSLagBoneControllers are only for skyrim and later, afaik
+        game = bpy.context.scene.niftools_scene.game
+        if game not in ('SKYRIM'):
+            return
+
+        n_controller = block_store.create_block("BSLagBoneController")
+        n_controller.name = block_store.get_full_name(b_bone)
+        #This should be active and cycle, what I've seen in game.
+        n_controller.flags = b_bone.niftools.lag_bone.flags #0x0048? hexa?
+        n_controller.frequency = 1.0
+        n_controller.phase = 0.0
+        n_controller.start_time = consts.FLOAT_MIN
```

Comment:
Are you sure these two aren't reversed? Both in the xml and the (admittedly single) BSLagBoneController I checked start_time is FLOAT_MAX and stop_time is FLOAT_MIN.